docs/docs/dev_docs/contracts/syntax/storage/storage_slots.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Storage slots
+---
+
+From the description of [storage slot in concepts](./../../../../concepts/foundation/state_model/storage_slots.md) you will get an idea around the logic of storage slots. In this section we will go into more detail and walk through an entire example of how storage slots are computed for private state to improve our storage slot intuition. Recall, that storage slots are only a logical construct in the private domain, and are not "actually" used for lookups, but rather just as a value to constrain against.
```

Comment:
> Recall, that storage slots are only a logical construct in the private domain

This could be misread as storage slots are only a 'thing' in the private domain; and not used in the public domain.

Perhaps "Recall, that in the private domain, storage slots are only a logical construct...